guides/common/modules/proc_synchronizing-content-over-http-or-https.adoc

```diff
@@ -0,0 +1,26 @@
+[id="synchronizing-content-over-http-or-https_{context}"]
+= Synchronizing Content Over HTTP or HTTPS
+
+Synchronize repositories on a downstream {ProjectServer} from a connected upstream {ProjectServer} over HTTP or HTTPS.
+
+.Prerequisites
+* A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.
+* You have imported the subscriptions manifest on both the upstream and downstream {ProjectServer}.
+* The required repository is enabled for the organization on the upstream {ProjectServer}.
+* You must have permissions to create Product repositories and organizations on the downstream {ProjectServer}.
+
+.Procedure
+. Navigate to *Content* > *Subscriptions*.
+. Click the *Manage Manifest* button.
+. Navigate to the *CDN Configuration* tab.
+. Select the *Upstream {ProjectServer}* tab.
+. In the *URL* field, enter the address of the upstream {ProjectServer}.
+. In the *Username*, enter your username for upstream login.
+. In the *Password*, enter your password or personal access token for upstream login.
+. In the *Organization label* field, enter a label for the downstream organization.
+. Optional: In the *Lifecycle Environment Label* field, enter a label for the downstream lifecycle environment.
+. Optional: In the *Content view label* field, enter a label for the downstream Content View.
+. From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
+. Navigate to *Content* > *Products*.
+. Select repositories to synchronize.
+. Click *Sync Now*.
```

Comment:
@jturel I was wondering what happens with the Sync Plan of the synced repo -- is it copied from the upstream server as well?

Reply:
There's no connection to any sync plans from the upstream server providing the enabled repo - that would need to be created by hand. And that makes sense - the downstream server really just wants the repo content available and any sync plans can be made specifically for that environment